libs/ui/selection-list/src/lib/selection-list/selection-list.component.ts

```diff
@@ -148,6 +148,11 @@ export class TsSelectionListComponent implements
   OnDestroy,
   TsFormFieldControl<unknown> {
 
+  /**
+   * Determine if the dropdown arrow icon should be visible
+   */
+  public showArrow;
```

Comment:
We should probably add a default value (or type) to this property.

Reply:
Added a type because there shouldn't be a default value... it should be based on whether there's option list or not.

Reply:
If we're dealing with booleans, not adding it is implicitly setting the default of `false`. But as long as there's a type it's still clear imo